frontend/src/pages/projects/pvc/MountPathField.tsx

```diff
@@ -50,10 +50,11 @@ const MountPathField: React.FC<MountPathFieldProps> = ({
     </InputGroup>
     <FormHelperText>
       <HelperText>
-        <HelperTextItem variant={mountPath.error ? 'error' : 'default'}>
-          {mountPath.error
-            ? 'Enter a path to a model or folder. This path cannot point to a root folder.'
-            : 'Must consist of lower case letters and dashes.'}
+        <HelperTextItem
+          variant={mountPath.error ? 'error' : 'default'}
+          data-testid="mount-path-folder-helper-text"
+        >
+          {mountPath.error ? mountPath.error : 'Must consist of lower case letters and dashes.'}
```

Comment:
nit, can be simplified to:
```suggestion
          {mountPath.error || 'Must consist of lower case letters and dashes.'}
```

Reply:
Updated.